package.json

```diff
@@ -1,16 +1,16 @@
 {
   "name": "elixir-ls",
-  "displayName": "ElixirLS Fork: Elixir support and debugger",
+  "displayName": "ElixirLS: Elixir support and debugger",
   "homepage": "https://github.com/elixir-lsp/elixir-ls",
   "repository": {
     "type": "git",
     "url": "https://github.com/elixir-lsp/vscode-elixir-ls.git"
   },
   "icon": "images/logo.png",
   "description": "Elixir support with debugger, autocomplete, and more. Powered by ElixirLS.",
-  "author": "Jake Becker",
+  "author": "The Elixir community",
```

Comment:
Haha, just realized that we probably should have changed this to something earlier :sweat_smile: